frontend/src/metabase/core/components/Select/Select.jsx

```diff
@@ -206,57 +193,64 @@ export default class Select extends Component {
       .filter(n => n);
 
     return (
-      <PopoverWithTrigger
-        ref={ref => (this._popover = ref)}
-        triggerElement={
-          this.props.triggerElement || (
-            <SelectButton
-              ref={this.selectButtonRef}
-              className="flex-full"
-              hasValue={selectedNames.length > 0}
-              disabled={disabled}
-              {...buttonProps}
-            >
-              {selectedNames.length > 0
-                ? selectedNames.map((name, index) => (
-                    <span key={index}>
-                      {name}
-                      {index < selectedNames.length - 1 ? ", " : ""}
-                    </span>
-                  ))
-                : placeholder}
-            </SelectButton>
-          )
-        }
-        onClose={composeEventHandlers(onClose, this.handleClose)}
+      <TippyPopoverWithTrigger
+        placement="bottom-start"
+        onHide={onClose}
+        renderTrigger={({ onClick }) => {
+          return (
+            this.props.triggerElement || (
+              <SelectButton
+                ref={this.selectButtonRef}
+                className="flex-full"
+                hasValue={selectedNames.length > 0}
+                disabled={disabled}
+                {...buttonProps}
+                onClick={onClick}
+              >
+                {selectedNames.length > 0
+                  ? selectedNames.map((name, index) => (
+                      <span key={index}>
+                        {name}
+                        {index < selectedNames.length - 1 ? ", " : ""}
+                      </span>
+                    ))
+                  : placeholder}
+              </SelectButton>
+            )
+          );
+        }}
+        popoverContent={({ closePopover, maxHeight }) => (
+          <AccordionList
+            maxHeight={maxHeight}
```

Comment:
@WiNloSt I faced a similar issue before and I believe it could be fixed by changing how `AccordionList` scrolls to a selected item (row). When determining the bounds of what's visible on the screen, it could find the scroll container that actually overflows instead of statically relying on `this.container`. WDYT?

Reply:
@ranquild Is there a reference for the said issue I could take a look at? But sounds good to try that as I thought passing `maxHeight` felt really hacky. Or at least so convenient.

Reply:
@WiNloSt Unfortunately I don't have a reference to the issue because it was years ago.